changes/286.general.rst

```diff
@@ -0,0 +1,3 @@
+Add clip_accum parameter to alignment methods.
```

Comment:
I am not quite sure of the correct formatting for this, but I think there should be a `[tweakreg]` at the beginning of each line to ensure the change log update goes under the correct header.  You can verify with @zacharyburnett to make sure the formatting for these fragments.

Reply:
there does not *need* to be (`towncrier` only looks at the filename when determining the header), however, adding `[step]` makes it a bit clearer what it's doing

Reply:
this is how it looks at the moment:
[<img width="550" alt="image" src="https://github.com/user-attachments/assets/5bff96c4-c556-4bbb-83bc-8f285f7185f9">](https://stcal--286.org.readthedocs.build/en/286/stcal/changes.html)